static/js/index.js

```diff
@@ -171,8 +174,31 @@ ep_comments.prototype.init = function(){
     saveItemsData: this.saveCommentWithoutSelection.bind(this),
     saveSubItemsData: this.saveRepliesWithoutSelection.bind(this),
   });
+
+  linesChangedListenerDetailed.onLineChanged('*', function(linesChanged) {
+    self._setLineOfChange(linesChanged);
+    self.updateIconsPositionSchedule.padChanged();
```

Comment:
Entendi pq vc guardou o número da linha alterada em uma prop do `this.textMarkIconsPosition`: a hora que vc detecta a mudança e a hora de processar essa mudança são diferentes! Blz, faz sentido. Eu só não guardaria o valor no `this.textMarkIconsPosition` -- guardaria no `this` mesmo, que é quem sabe dessa "assincronia".

Eu explico melhor o pq nos comentários no `this.textMarkIconsPosition`, logo abaixo